code/__DEFINES/move_force.dm

```diff
@@ -7,4 +7,17 @@
 #define MOVE_FORCE_PULL_RATIO 1				//Same move force to pull objects
 #define MOVE_FORCE_PUSH_RATIO 1				//Same move force to normally push
 #define MOVE_FORCE_FORCEPUSH_RATIO 2		//2x move force to forcefully push
-#define MOVE_FORCE_CRUSH_RATIO 3			//2x move force to do things like crush objects
+#define MOVE_FORCE_CRUSH_RATIO 3			//3x move force to do things like crush objects
+
+#define MOVE_FORCE_RESIST_MEGAFAUNA MOVE_RESIST_DEFAULT * MOVE_FORCE_CRUSH_RATIO * 11
+#define MOVE_FORCE_RESIST_STATUE MOVE_RESIST_DEFAULT * MOVE_FORCE_CRUSH_RATIO * 11
+#define MOVE_FORCE_RESIST_GOLIATH MOVE_RESIST_DEFAULT * MOVE_FORCE_CRUSH_RATIO * 6
+#define MOVE_FORCE_RESIST_SPAWNER MOVE_RESIST_DEFAULT * 30

```

Comment:
I'm not really a fan of where you're going with this. Defines for move resist is fine but name these a bit more generic like `MOVE_RESIST_WEAK` etc

Reply:
kay